Makefile

```diff
@@ -2,16 +2,16 @@
 .PHONY: build clean sdist test
 
 build:
-	pip3 install -r requirements.txt
+	pipenv install
 
 clean:
 	rm -rf sdc/crypto/doc/html
 	rm -v dist/sdc-cryptography-*.tar.gz
 
 sdist:
-	python setup.py sdist
+	pipenv run python setup.py sdist
 
-test: build
-	pip3 install -r test_requirements.txt
-	flake8 .
-	python -m unittest discover sdc
+test:
+	pipenv install --dev
+	pipenv run flake8 .
```

Comment:
Would prefer these to be bare commands, e.g. `flake8 .`

That way pipenv is not required and developer can use whichever virtualenv solution they prefer, or just run `pipenv run make test`